R/trace_generation.R

```diff
@@ -358,8 +363,7 @@ toBasic <- list(
     g
   },
   bar=function(g) {
-    if (any(is.na(g$prestats.data$x)))
-      g$prestats.data$x <- g$prestats.data$x.name
+    ## TODO: why is this here? bar is a basic geom.
```

Comment:
I guess a 'git blame' would help tracing this back.
I agree it looks non-robust.  I remember doing this so that popular use cases convert well...